docs/source/el7toel8/deprecation.md

```diff
@@ -0,0 +1,27 @@
+# Deprecated functionality in the el7toel8 repository
+
+Here is the list of deprecated functionality in the el7toel8 repository. Each
+version section contains just the newly introduced deprecation. The dates
+in brackets correspond to the end of the deprecation protection period,
+after which the related functionality can be whenever removed.
```

Comment:
"after which the related functionality can be removed at any time."

Reply:
Done.